fs2/src/main/scala/laserdisc/fs2/LoggingSystem.scala

```diff
@@ -0,0 +1,20 @@
+package laserdisc
+package fs2
+
+import cats.Applicative
+import log.effect.LogWriter
+import log.effect.fs2.SyncLogWriter
+
+private[fs2] class LoggingSystem[F[_]](private[fs2] val ls: LogWriter[F]) extends AnyVal
```

Comment:
this in itself might be worth extracting out to the logging project as a separate artifact lib?

Reply:
It could yes, let me think to a way to scope it into `log-effect-fs2`. I have to find good naming. Here is an internal component but as part of a library I want to find the right one. 👍